### ISSUE TITLE
I followed all the steps, but the bot is not working. Don't know what the problem is.

### ISSUE THREAD

**Issue by tgurgel27** [OfficeDev/microsoft-teams-faqplusplus-app#76]

**Comment by tgurgel27** [comment]
The app was created successfuly but the bot doesn't seem to be working

![QnA Plus error](https://user-images.githubusercontent.com/55068929/64522326-22c6df80-d2d0-11e9-8806-e5321e76de76.jpg)

**Comment by GeorgianaMihaela** [comment]
Hello @tgurgel27 

In your print screen, I see that you do not receive reply from the bot in teams. 

I would have a few suggestions:
1. what is the outcome if you press on + to create a new chat with the bot and paste the bot ID? Do you get reply? 

2. If you still do not, could you have a look at the suggestions in this issue
https://github.com/OfficeDev/microsoft-teams-faqplusplus-app/issues/78? 

Basically, to troubleshoot, you would need to download the code, run it locally, test in the emulator (debug it actually) and understand where it stops. 

Hope this helps.

**Comment by jsuddsjr** [comment]
After debugging the bot code locally, I always get a 401 error when it processes the request. Do we need an auth header, and how do I ensure the bot is authenticated?
